std/concurrency.d

```diff
@@ -765,6 +747,42 @@ do
         return ret;
 }
 
+///
+@system unittest
+{
+    auto tid = spawn(
+    {
+        assert(receiveOnly!int == 42);
```

Comment:
I agree that this way is cleaner, but how do you ensure that the `assert` was actually called?

Reply:
That's an example, the focus is on good looking code. We can add a non-documented real test if necessary.